dashboard/app/helpers/survey_results_helper.rb

```diff
@@ -1,6 +1,7 @@
 module SurveyResultsHelper
   DIVERSITY_SURVEY_ENABLED = false
-  NPS_SURVEY_ENABLED = false
+  DEFAULT_NPS_AUDIENCE = "none"
+  NPS_AUDIENCE = DCDO.get('nps_audience', DEFAULT_NPS_AUDIENCE)
```

Comment:
Rails conventions use capitalized variable names for constants.  This isn't a constant so it should probably be called nps_audience instead.  Also, we should move this into show_nps_survey? so it has local scope.